src/Sylius/Bundle/AdminBundle/Form/Type/TranslatableAutocompleteType.php

```diff
@@ -112,7 +112,7 @@ private function normalizeFilterQuery(Options $options, ?callable $filterQuery):
     private static function getComparisons(Expr $expr, array $fields): iterable
     {
         foreach ($fields as $field) {
-            yield $expr->like($field, ':query');
+            yield $expr->like($field, 'LOWER(:query)');
```

Comment:
It won't work this way. Scenarios need adjusting, because in the setup you set a specific name, and then this logic:
https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Behat/Context/Setup/TaxonomyContext.php#L154
parses the name into a code. In practice, you search by code, not by the translatable name:
https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/AdminBundle/Form/Type/TranslatableAutocompleteType.php#L92

Add taxons that have different codes and translatable names to reproduce it.

 
```suggestion
            yield $expr->like((string) $expr->lower($field), 'LOWER(:query)');
```

Reply:
done